scripts/linters/pylint_extensions.py

```diff
@@ -42,7 +42,7 @@
 # the punctuation and capital letter checks will be skipped for that
 # comment or docstring.
 EXCLUDED_PHRASES = [
```

Comment:
I've checked we can't move it to .pylintrc. It didn't worked there.

Reply:
> I've checked

I'm not sure what you have checked! Can you please share the doc where it's written "we can't move it to .pylintrc."?

How does we configure other checkers using the .pylintrc file?

Reply:
You can check here, .pylintrc file is used to configure properties not for adding variables or constants to it.

Reply:
Yeah, we are configuring the linter here. How do we have max-len=80, regex-patterns for function name etc? I think there must be some way to do it. 

This is important because it will allow us to re-use these custorm_linters in other oppia projects. We just have to change the .pylintrc file!

Reply:
I tried but didn't find the option for type: list option. Can you  help me here?

Reply:
Marking this resolved (As per our offline discussions!)